### PR TITLE
Fix some styling on the showcase page

### PR DESCRIPTION
**Pull request by alexandernanberg**
**What**

Some style fixes to the showcase page
- Align icons properly
- Fix overlapping filter nav (see screenshot)

**Before and after shots**

image

We probably want to add some more styling to to search input and some other areas, might create a new PR for that

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit c210e79fe81be25a59d89b9f9a1784a034b9e1d5

https://app.netlify.com/sites/using-postcss-sass/deploys/5b65949e73f2cf21404f76cf

File: www/src/views/shared/styles.js [hunk 1]
```diff
@@ -132,6 +132,10 @@ const styles = {
     borderBottom: `none !important`, // i know i know
     boxShadow: `none !important`, // but people really want this
   },
+  searchInput: {
+    paddingLeft: `1.4rem`,
+    fontFamily: typography.options.headerFontFamily.join(`,`),
```
[the remark this answers is not in the document]
Comment (alexandernanberg) [reply www/src/views/shared/styles.js]: Sounds good, done that now! 👍

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 5ca1f5e732d43289252b58e80b78ed727b98aa98

https://app.netlify.com/sites/gatsbyjs/deploys/5b6f4eadb13fb1133f3b7f0d

**Comment by KyleAMathews** [comment]
Deploy preview for *using-remark* failed.

Built with commit 6b00fdabd90e1b4a778078765a2affe9a10ad585

https://app.netlify.com/sites/using-remark/deploys/5b71739dc965925091c6961e

**Pull request merged**